### ISSUE TITLE
JetSmartFilters -> "Search" Filter : make search by terms possible

### ISSUE THREAD

**Issue by cookiemachine** [CrocoBlock/suggestions#7664]
desired feature for the "Search" filter (SmartFilter):
- allow search by terms/taxonomies when search by "custom field"
- at the same time, also allow search by meta fields

![search_01](https://github.com/user-attachments/assets/3ef1cd5e-79f0-46dd-9e36-5f95ed6fe153)

**EXAMPLE:**

Example CPT:
meta field: Type
- meta field value: house

Taxonomy: color
- term: blue

Example Smartfilter "Search":
it queries for the meta field "type" and the taxonomy "color"

Example Front end:
- user enters "blue house" into the "search" Smartfilter
- result: our above example CPT appears. (It would also appear, if only "blue", "house", "blu" etc. was entered of course)

**Basically: Please make it possible to use the JetSmartFilter "Search" for terms.**

**Comment by rob1998** [comment]
I second this, I have a search filter and would like for it to allow users to type in a search prompt
which would then search through a custom meta field description, a taxonomy municipality and a taxonomy city/town
This would allow users to easily search the CPT items without having to know the structure of the CPT and having to use multiple kinds of filters.
